apps/wakucanary/wakucanary.nim

```diff
@@ -190,6 +205,13 @@ proc main(rng: ref HmacDrbgContext): Future[int] {.async.} =
 
   var enrBuilder = EnrBuilder.init(nodeKey)
 
+  let relayShards = RelayShards.init(conf.clusterId, conf.shards)
+  if relayShards.isok():
+    enrBuilder.withWakuRelaySharding(relayShards.get()).isOkOr:
+      error "Building ENR with relay sharding failed", error = error
+  else:
+    error "Relay shards initialization failed", error = relayShards.error
```

Comment:
Just a nitpick comment where we use `valueOr` and, in case of error, we stop the `wakucanary` execution.

Notice how `valueOr` is very similar to `isOkOr`, in concept, but in this case, it allows to return an item.
```suggestion
  let relayShards = RelayShards.init(conf.clusterId, conf.shards).valueOr:
    error "Relay shards initialization failed", error = error
    return 1

  enrBuilder.withWakuRelaySharding(relayShards.get()).isOkOr:
    error "Building ENR with relay sharding failed", error = error
    return 1
```